tests/js/spec/views/eventsV2/results.spec.jsx

```diff
@@ -16,7 +16,7 @@ const FIELDS = [
     field: 'user',
   },
   {
-    field: 'count(user)',
+    field: 'count(id)',
```

Comment:
You got rid of `count(id)` everywhere else, why not here?

Reply:
Fixed now.